cirq-google/cirq_google/api/v2/sweeps_test.py

```diff
@@ -134,3 +134,37 @@ def test_sweep_with_flattened_sweep():
     param_sweep1 = cirq.Linspace('t', start=0, stop=1, length=20)
     (_, param_sweep2) = cirq.flatten_with_sweep(circuit, param_sweep1)
     assert v2.sweep_to_proto(param_sweep2) is not None
+
+
+def test_run_context_to_proto():
```

Comment:
Can you also add a test to demonstrate that specifying the `out` parameter is appended to? Same for the batch case.

Reply:
Done.